### PR TITLE
chore: release v4.2.0

### PR DESCRIPTION
**Pull request by fzipi**
The latest and greatest v4.2.0

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -8,6 +8,18 @@
...
+Changes with direct rule impact (sorted by lowest rule ID per change where available):
+
+ * fix: increase length of Accept-Encoding header from 50 to 100 (920520 PL1) (Franziska Bühler) [#3661]
+ * fix: add detection for php evasion attempt (933100 PL1) (Franziska Bühler) [#3667]
+ * fix: add visudo and cscli to unix-shell.data (932160 PL-1, 932161 PL-2) (@EsadCetiner) [#3663]
+
+Changes without direct rule impact:
+
+ * feat: disassemble php rule (933100 PL1) (Franziska Bühler) [#3662]
+
```
[the remark this answers is not in the document]
Comment (fzipi) [reply CHANGES.md:22]: I'm just pasting the contents of the .changes-pending.md here. So if changes are lost from the changelog, it is out of my scope.

@dune73 @theseion ☝️ ?

[the remark this answers is not in the document]
Comment (fzipi) [reply CHANGES.md:22]: Trying to follow up changes in the `.changes-pending.md` file with `git log --follow -p -- .changes-pending.md` gives me what is there. 🤷

File: util/crs-rules-check/examples/test8.conf [hunk 2]
```diff
@@ -4,23 +4,23 @@ SecRule &TX:blocking_paranoia_level "@eq 0" \
     phase:1,\
     pass,\
     nolog,\
-    ver:'OWASP_CRS/4.0.0-rc1',\
```
Comment (theseion) [inline util/crs-rules-check/examples/test8.conf:7]: These are probably unintended changes.

**Review by theseion — CHANGES_REQUESTED** [review]

File: util/crs-rules-check/examples/test8.conf (hunk 2 above)
Comment (fzipi) [reply util/crs-rules-check/examples/test8.conf:7]: Yeah, but in the end it always pops up, so 🤷

Comment (theseion) [reply util/crs-rules-check/examples/test8.conf]: Probably need a way to exclude directories from the update.

**Review by theseion — APPROVED** [review]

**Review by franbuehler — APPROVED** [review]
LGTM. Thank you!!

**Pull request merged**